docs/build/build-parachains.md

```diff
@@ -87,7 +87,7 @@ With {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, you will ge
 of the box if you build your application as a parachain. The [XCM format](../learn/learn-xcm.md)
 allows any parachains to communicate by passing messages between them. Furthermore, as bridges to
 other chains are connected (such as those to Bitcoin or Ethereum)
-{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} parachains will be able to
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s parachains will be able to
 communicate with these as well.
```

Comment:
Interestingly these `'` in conditional rendering brackets cause the syntax highlighting in vscode to break.  This is an easy fix for a simple case.  I haven't encountered this in other areas of the docs yet.

Reply:
Will keep this in mind while reviewing the PRs. Thanks!